include/glow/Backend/Backend.h

```diff
@@ -128,8 +143,12 @@ class Backend {
 };
 
 /// Create a backend of kind \p kind.
+/// Deprecated. Moving gradualy to the createBackend based on the backend name.
```

Comment:
So we won't keep this interface in the future. Is that due to code complexity or others? I thought we would support both : from Backend Name and Backend Kind

Reply:
BackendKind will go away. It's not practical to keep that enum up to date with auto registration of backends.

In the following PRs each backend will have static a method `getName()` which will return name of that backend and essentially will be used as a key everywhere. There will be no need for the enum BackendKind.